src/chessBoardCornersDetector.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <opencv2/opencv.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <Chessboard.h>
+using namespace std;
+using namespace cv;
+
+
+cv::Size boardSize;
+
+Result chessBoardCornersDetector(std::string image_path, int board_width, int board_height,const bool verbose){
```

Comment:
I think you should structure your function such that it takes a list of image paths and then runs corner detection for all of them and return the corners in the same order.